oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -74,8 +74,8 @@ impl NodeStopper {
             .send(())
             // Notification errors are discarded since not all of the Nodes save
             // and use the [`oneshot::Receiver`].
-            .unwrap_or_else(|error| {
-                warn!("Couldn't send notification to {}: {:?}", node_name, error);
+            .unwrap_or_else(|_| {
```

Comment:
Could you make it clear that this is just discarding a unit value every time at most?

```suggestion
            .unwrap_or_else(|()| {
```

Reply:
Done.